src/helpers/BoxHelper.js

```diff
@@ -45,7 +45,11 @@ BoxHelper.prototype.update = ( function () {
 
 		}
 
-		if ( this.object !== undefined ) {
+		if ( this.object instanceof Box3 ) {
```

Comment:
BTW: We now use the `is*` properties like `isBox3` to determine the type. See https://github.com/mrdoob/three.js/blob/dev/src/math/Box3.js#L18